graphing/src/cloudevolution/views.py

```diff
@@ -94,19 +95,71 @@ def expt_name(request, experiment):
 
 	return render(request, "experiment.html", context)
 
+
 def dilutions(request, experiment):
 	sidebar_links, subdir_log = file_scan('expt')
 	vial_count = range(0, 16)
 	expt_dir, expt_subdir = file_scan(experiment)
 	rootdir = os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))))
 	evolver_dir = os.path.join(rootdir, 'experiment')
 	pump_cal = os.path.join(evolver_dir, expt_subdir[0], "pump_cal.txt")
+	bottle_file = os.path.join(evolver_dir, expt_subdir[0], "bottles.txt")
```

Comment:
Where should we place the `bottles.txt` file?
- `TestExperiment/bottles.txt`
- `TestExperiment/TestExperiment_expt/bottles.txt`